pilot/pkg/networking/core/cluster_builder.go

```diff
@@ -344,8 +344,13 @@ func (cb *ClusterBuilder) buildCluster(name string, discoveryType cluster.Cluste
 	if direction == model.TrafficDirectionOutbound {
 		// If stat name is configured, build the alternate stats name.
 		if len(cb.req.Push.Mesh.OutboundClusterStatName) != 0 {
```

Comment:
The other option is to set the default if `cb.req.Push.Mesh.OutboundClusterStatName` is not present

Reply:
There's also InboundClusterStatName to consider so I just decided to set the alt stat name by default